docs/HACKING_QUICKSTART.md

```diff
@@ -66,6 +66,8 @@ If you build with `-d`, run with `-d`. If you build with `-r`, run with `-r`.
 ## mach
 
 `mach` is a python utility that does plenty of things to make our life easier (build, run, run tests, update dependencies… see `./mach --help`). Beside editing files and git commands, everything else is done via `mach`.
+In most cases, you can also use `cargo` to build and run `servo`. Common cases where pure `cargo build` does not 
```

Comment:
I'm not sure if we want to officially recommend using `cargo` just yet. There are other cases such as building with GStreamer support that also require using `mach`. @mrobinson, wdyt?

Reply:
Okay, I dropped the documentation changes for now.

Reply:
I think it is still helpful to have the text about enabling `crown` when hacking on DOM. 
With the current documentation, we recommend enabling it for NixOS, but not on other platforms where I believe we can update `rust-analyzer.check.overrideCommand`  to pass `--use-crown` to `mach check`.

Reply:
Good point - I updated the documentation to now mention `--use-crown` and how to setup rust-analyzer for vs-code to do the same.